docs/content/docs/spark/spark-procedures.md

```diff
@@ -340,6 +354,15 @@ CALL catalog_name.system.rewrite_manifests('db.sample', false)
 
 The `snapshot` and `migrate` procedures help test and migrate existing Hive or Spark tables to Iceberg.
 
+{{< hint info >}}
```

Comment:
We should be able to remove this warning now that the fix is released, right?

Reply:
yes you are right, updated